app/views/articles/_full_comment_area.html.erb

```diff
@@ -2,10 +2,28 @@
   <section id="comments" data-follow-button-container="true" data-updated-at="<%= Time.current %>" class="text-padding mb-4 border-t-1 border-0 border-solid border-base-10">
     <% if @article.show_comments %>
       <header class="relative flex justify-between items-center mb-6">
-        <h2 class="crayons-subtitle-1">
-          <%= t("views.articles.comments.subtitle_html",
+        <h2 class="crayons-subtitle-1" id="toggle-comments-sort-dropdown" aria-controls="comments-sort-dropdown" aria-expanded="false" aria-label="Toggle Comment Sort Order dropdown menu" aria-haspopup="true">
```

Comment:
Refactor (blocking): A header cannot be used as the trigger for the dropdown as it causes accessibility issues. A sort button to the right of the header as the trigger will be more accessible. cc: @aitchiss

Reply:
Checking out the designs on https://github.com/forem/forem/issues/16133, I think you can use an [icon only button](https://storybook.forem.com/?path=/story/components-buttons--icon-only) with the `expand.svg` icon.

In terms of accessibility, be sure to give the button an `aria-label` that describes the function ("sort comments" would work). We also have a helper to initialize dropdown expand/collapse behaviours in JS (incuding accessibility features) - check out https://github.com/forem/forem/blob/main/app/javascript/utilities/dropdownUtils.js#L118